lib/global-styles.php

```diff
@@ -492,6 +492,16 @@ function gutenberg_experimental_global_styles_resolver_styles( $block_selector,
 	$css_rule         = '';
 	$css_declarations = '';
 
+	// Transformations for block-supports strings.
+	foreach ( $block_supports as $block_support ) {
+		// Convert camelCase to kebab-case and add to array.
+		$block_supports[] = strtolower( preg_replace( '/([a-z0-9]|(?=[A-Z]))([A-Z])/', '$1-$2', $block_support ) );
+	}
+	// When adding the kebab-case supports, we didn't check if the item exists in the array
+	// to avoid constly in_array() calls. We'll need to remove the duplicates here.
```

Comment:
What is `constly` ?

Reply:
LOL it was a typo (costly).
Changed the wording of these comments, I hope it's cleaner now.
Thank you :)